deploy/hawtio-embedded/pom.xml

```diff
@@ -25,14 +25,26 @@
       <type>war</type>
     </dependency>
     <dependency>
-      <groupId>org.eclipse.jetty</groupId>
-      <artifactId>jetty-webapp</artifactId>
-      <version>${jetty-version}</version>
+      <groupId>io.hawt</groupId>
+      <artifactId>hawtio-system</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.hawt</groupId>
+      <artifactId>hawtio-local-jvm-mbean</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
```

Comment:
are these `test` intentional?  why do we introduce them?

Reply:
without these, `io.hawt.embedded.MainTest` doesn't work in IDE and works only with `mvn clean install`, because it requires packaged jar artifact. `test` scope is safe - I checked that it doesn't affect the target artifact.